### DISCUSSION TITLE
Bring into core?

### DISCUSSION THREAD

**Discussion by krossekrabbe** [AvaiBookSports/DoctrineMigrationsMultipleDatabaseBundle#17] [General]
Hi,

thank you for your awesome work, I was astonished when I found out this is not working out of the box with doctrine-migrations. For me it seems the namespaced migration feature does not make too much sense without either allowing to configure the entity manager per namespace or allowing to pass the namespace + em to the migrate command :thinking: I ended up having multiple EMs and connections defined + migration namespaces... and then `d:m:m` happily ignores that and runs all migrations of all namespaces against the default connection :eyes: Feels like half of the feature got missing :) 

Have you thought about bringing the adjustments you have made into doctrine migrations core?

**Comment by devnix** [comment]
Hi @krossekrabbe
> and then `d:m:m` happily ignores that and runs all migrations of all namespaces against the default connection eyes

I'm not sure about what you have set up. We have been using this package for 2+ years and haven't observed an issue like that (nor have other users from this package). Please try replicating the problem in a repository with instructions to see what's happening.

> Have you thought about bringing the adjustments you have made into doctrine migrations core?

I would ping at https://github.com/doctrine/DoctrineMigrationsBundle/issues/38. I don't have the time/knowledge to do a PR that probably would not get merged. I think it's more important to have a conversation with the maintainers of the project to know if they want to support a feature like this.

**Comment by krossekrabbe** [comment]
Hi @devnix 

> and then d:m:m happily ignores that and runs all migrations of all namespaces against the default connection eyes

> I'm not sure about what you have set up. We have been using this package for 2+ years and haven't observed an issue like that (nor have other users from this package). Please try replicating the problem in a repository with instructions to see what's happening.

Misunderstanding, your bundle works perfectly fine! I was talking about the original doctrine-migrations commands which are missing the functionality this bundle provides.
